### ISSUE TITLE
Files not saved to working directory for Watertight Geometry workflow

### ISSUE THREAD

**Issue by cj-hodgson** [ansys/pyfluent#1393]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

When the Watertight Geometry meshing workflow is used in pyfluent files are not saved to the users working directory, and are saved to the user's temporary directory instead. It is not obvious where the files have been written.
The equivalent script run within Fluent writes files to the working directory, as does a pyfluent script that executes a Fault-tolerant Meshing workflow. 

### 📝 Steps to reproduce

To reproduce, please run the script mixing_elbow_pyfluent.py --> the files mixing_elbow_pyfluent.msh.h5 and mixing_elbow1_pyfluent.cas.h5 are not written to the user's working directory.

Also attached are scripts to run the same case in Fluent pyconsole (mixing_elbow_pyconsole.py) and an example using the Fault-tolerant Meshing workflow in pyfluent (exhaust_system_pyfluent.py).

[pyfluent_watertight_geometry.zip](https://github.com/pyansys/pyfluent/files/10822891/pyfluent_watertight_geometry.zip)

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

2023 R2

### 🐍 Which Python version are you using?

3.10

### 📦 Installed packages

```shell
ansys-api-fluent==0.3.7
ansys-api-platform-instancemanagement==1.0.0b3
ansys-fluent-core==0.13.dev2
ansys-fluent-parametric==0.7.dev0
ansys-fluent-visualization==0.7.dev0
ansys-platform-instancemanagement==1.0.3
appdirs==1.4.4
certifi==2022.12.7
charset-normalizer==3.0.1
contourpy==1.0.7
cycler==0.11.0
fonttools==4.38.0
googleapis-common-protos==1.58.0
grpcio==1.52.0rc1
h5py==3.8.0
idna==3.4
imageio==2.25.1
importlib-metadata==6.0.0
kiwisolver==1.4.4
matplotlib==3.7.0
numpy==1.24.2
packaging==23.0
pandas==2.0.0rc0
Pillow==9.4.0
pooch==1.6.0
protobuf==3.20.3
protoc-gen-swagger==0.1.0
pyparsing==3.0.9
PySide6==6.4.2
PySide6-Addons==6.4.2
PySide6-Essentials==6.4.2
python-dateutil==2.8.2
pytz==2022.7.1
pyvista==0.38.2
pyvistaqt==0.9.1
QtPy==2.3.0
requests==2.28.2
scooby==0.7.1
shiboken6==6.4.2
six==1.16.0
urllib3==1.26.14
vtk==9.2.6
zipp==3.14.0
```

**Comment by seanpearsonuk** [comment]
`launch_fluent(cwd=...)` has no effect on the file write location. TUI logic is choosing the path itself if one is not specified in the API call. Is there anything else we can do other than just documenting that the user should specify the path @mkundu1

**Comment by mkundu1** [comment]
@seanpearsonuk @cj-hodgson This is a Fluent side behaviour. `meshing.workflow.TaskObject["Import Geometry"].Execute()` changes Fluent's working directory to the location of geometry file.

**Comment by raph-luc** [comment]
@seanpearsonuk @mkundu1 
Noticed the same issue here: all examples that I tried output files to folders which aren't obvious or clear to the user, nor does the user have any control over where they go currently (as far as I can tell).

Can also confirm that specifying `launch_fluent(cwd=...)` does not help.

Rather than changing the default behavior of Fluent, I suggested (see [fix/examples_save_path](https://github.com/pyansys/pyfluent/tree/fix/examples_save_path)) adding the functionality to src/ansys/fluent/core/examples/downloads.py to be able to specify the path where the example files will be downloaded to. Doing this, all the output files go where the user specifies. Using the example above, `save_path` can be specified as:
`import_filename = examples.download_file("mixing_elbow.pmdb", "pyfluent/mixing_elbow", save_path='.')`
or just ommitted.
